flask_dynamo/manager.py

```diff
@@ -148,6 +149,40 @@ def _get_ctx(app):
                 'flask-dynamo extension not registered on flask app'
             )
 
+    @staticmethod
+    def _init_session(app):
+        session_kwargs = {}
+        # Only apply if manually specified: otherwise, we'll let boto
+        # figure it out (boto will sniff for ec2 instance profile
+        # credentials).
+        if app.config['AWS_ACCESS_KEY_ID']:
```

Comment:
Why aren't all these values being grabbed safely with `get()`, instead of just the last two?